float64[3] posLoad
